contracts/cloud.yml

```diff
@@ -1198,41 +1198,45 @@ paths:
             $ref: '#/components/schemas/WritePrecision'
       responses:
         '204':
-          description: Write data is correctly formatted and accepted for writing to the bucket.
+          description: 'The sent data was correctly formatted and accepted for writing to the bucket. `204` doesn''t indicate a successful write operation since writes are asynchronous. See [how to check for write errors](###some-link###).'
         '400':
-          description: Line protocol poorly formed and no points were written.  Response can be used to determine the first malformed line in the body line-protocol. All data in body was rejected and not written.
+          description: Bad request. The error indicates the line protocol payload was malformed. The response body contains the first malformed line in the data. All payload data was rejected and not written.
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/LineProtocolError'
         '401':
-          description: Token does not have sufficient permissions to write to this organization and bucket or the organization and bucket do not exist.
+          description: |
+            Unauthorized. The error may indicate one of the following:
+              * The `Authorization: Token` header is missing or malformed.
+              * The API token value is missing from the header.
+              * The token does not have sufficient permissions to write to this organization and bucket.
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
-        '403':
-          description: No token was sent and they are required.
+        '404':
+          description: 'Not found. A requested resource was not found. The response body contains the requested resource type, e.g. `organization name` or `bucket`, and name.'
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
         '413':
-          description: Write has been rejected because the payload is too large. Error message returns max size supported. All data in body was rejected and not written.
+          description: Request entity too large. The payload exceeded the 50MB limit. All payload data was rejected and not written.
```

Comment:
`413` is handled by Nginx before it hits the application. Nginx is returning HTML instead of JSON as specified. I opened https://github.com/influxdata/idpe/issues/11615 to correct that. In the meantime, should we remove or correct the content spec here?

Reply:
Changes to the openapi spec are quick enough it seems like in this case especially (discussed fix has no eta), we can go ahead and update the spec to match the api's behavior.

Reply:
I believe OSS returns this error case as JSON, so :+1: to adding an HTML response case but I'd leave the JSON case as well.

Reply:
Does this limit exist in OSS? I haven't been able to hit one and I don't see a config property for it.

Reply:
There is a code path in OSS to detect a too-big batch and return the correct error code, but the detection depends on Go's built-in `ioutil.ReadAll` raising an error and I'm not sure what the upper limit is within that (or if there even is one).

Reply:
@danxmoran Does OSS have a token quota or would that be managed by a separate gateway?

>    "429":
      description: The token is temporarily over quota. The Retry-After header describes when to try the write again.
      headers:
        Retry-After:
          description: A non-negative decimal integer indicating the seconds to delay after the response is received.
          schema:
            type: integer
            format: int32